app/components/subject-picker/template.hbs

```diff
@@ -2,9 +2,8 @@
     {{#each selected as |subject|}}
         <span class="subject">
             {{#each subject as |segment|}}
-                <b>{{segment.text}}</b>
+                <div class="subject-container"><b class="subject-container-text">{{segment.text}} <button class="cancel-subject" aria-label="Remove subject" title="Remove" {{action 'deselect' subject}}><i class="fa fa-close"></i></button></b></div>
```

Comment:
Thanks for adding ARIA labels! Can you please move all hard-coded text into a translations file?